fix: calculating fields and typo in translation

Aragent opened the pull request:
## Changes / Comments
fix: calculating fields and typo in translation





## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/The%20One%20Ring/roll20-character-sheets) *Roll20 Internal Use only.*

mperes commented:
Hello! Your Pull Request includes changes to translation files. Unfortunately, we no longer accept translations submitted outside of Crowdin. To learn more about Crowdin and how to submit translations through Crowdin, please see our article on [How to Become a Translation Volunteer.](https://roll20.zendesk.com/hc/en-us/articles/360058423993-How-to-Become-a-Translation-Volunteer) Thanks for all you do!

Aragent commented:
This is getting overly complicated, one where do you tell us we have to use
Crowdin for translation, this pull request gets closed because I provided
english translation. The One RIng Character Sheet uses translation, I added
a new feature and provided a text when hovered over, to create that text
had to be added to translation.json and in the translation folder to
en.json as an example. this gets approved, and implemented on monday 18th.
now I fix a minor bug to sheet.html to fix calculating fields and to
translation.json minor typo of experiance to experience and also to
translations folder en.json experiance to experience. and this gets closed
on tuesday the 19th. I am totally unfamiliar with Crowdin however I see we
can translate a line. but where do we create a new line? (Personally this
may be great for the roll20 staff, but for those of us who just maintain
character sheets not so much) if you are requiring us to use (Crowdin) then
translation files should be hidden files or files we don't have access to.
also if you continue to require us to use Crowdin are we allowed to remove
all translations from the character sheet as we added translations to make
it nicer and easier for international members, prior to Crowdin
requirement. Crowdin requires a completely new system we must learn to keep
a Character Sheet up to date or for modification.

Please let me know.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Tue, Jan 19, 2021 at 6:10 PM Miguel Peres <notifications@github.com>
wrote:

> Hello! Your Pull Request includes changes to translation files.
> Unfortunately, we no longer accept translations submitted outside of
> Crowdin. To learn more about Crowdin and how to submit translations through
> Crowdin, please see our article on How to Become a Translation Volunteer.
> <https://roll20.zendesk.com/hc/en-us/articles/360058423993-How-to-Become-a-Translation-Volunteer>
> Thanks for all you do!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Roll20/roll20-character-sheets/pull/8106#issuecomment-763202264>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJHRL57BUCJLSLBQQKPKB3DS2YGPVANCNFSM4WI54QZQ>
> .
>
